actions/websocket_actions.jsx

```diff
@@ -182,9 +182,11 @@ export function unregisterPluginReconnectHandler(pluginId) {
     Reflect.deleteProperty(pluginReconnectHandlers, pluginId);
 }
 
-export function reconnect(includeWebSocket = true) {
+export function reconnect(includeWebSocket = true, init = false) {
     if (includeWebSocket) {
         reconnectWebSocket();
+    } else if (init) {
+        initialize();
```

Comment:
Any reason we are not calling `reconnectWebSocket()`? It does close() and initialize() which seems like a clean way to achieve this.

Reply:
Ok, not a big deal either ways. Closing and creating a new connection is cheap, and you don't lose any events.

Reply:
Looking through the code, we never actually call `reconnect` without passing false for `includeWebsocket`. That must be left over from the mobile app. Since that's the case, could we just remove `reconnectWebSocket` and entirely and just have the new argument be the only one that `reconnect` has?

Reply:
oh, you are correct, OK I'll apply @agnivade's suggestion and just use `reconnectWebSocket` instead of replacing the option entirely.